decidim-dev/lib/decidim/dev/test/rspec_support/screenshot_helper.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+#
+# Copyright (c) 2018 David Rodríguez - The MIT License (MIT)
+# Originally copied from https://gitlab.com/deivid-rodriguez/system_test_html_screenshots
+#
+
+require "action_dispatch/system_testing/test_helpers/screenshot_helper"
+
+module ActionDispatch
+  module SystemTesting
+    module TestHelpers
+      module ScreenshotHelper
```

Comment:
We will need to reflect the changes at #8802 also to this class.

Please wait first that #8802 is merged and that override can be then removed as it will be moved to this file.